estudios_socioeconomicos/serializers.py

```diff
@@ -63,3 +68,101 @@ class Meta:
             'nombre',
             'numero',
             'subsecciones')
+
+
+class RespuestaSerializer(serializers.ModelSerializer):
+    """ Serializer for using .models.Subseccion objects
+        in REST endpoint.
+    """
+    class Meta:
+        model = Respuesta
+        fields = (
+            'estudio',
+            'pregunta',
+            'eleccion',
+            'respuesta')
+
+        read_only_fields = ('estudio', )
+
+
+class EstudioSerializer(serializers.ModelSerializer):
+    """ Serializer to represent a .models.Comentario instance
+        through a REST endpoint for the offline application
+        to submit information.
+    """
+    respuesta_estudio = RespuestaSerializer(many=True, allow_null=True)
+    familia = FamiliaSerializer()
+
+    class Meta:
+        model = Estudio
+        fields = (
+            'id',
+            'familia',
+            'status',
+            'capturista',
+            'respuesta_estudio')
+
+        read_only_fields = ('id', 'status', 'capturista')
+
+    def create(self, capturista):
+        """ This function overides the default behaviour for creating
+            an object through a serializer.
+
+            Django Rest Framework does not implement saving nested
+            serializers.
+
+            Estudio object depends on a Familia insance. This function
+            first creates the familia, then creates the Estudio and
+            finally creates all answers that depend on the Estudio.
+
+            Notes
+            -----
+            Saves the Estudio instance using .save_base(raw=True). This
+            is required because Estudio has a Trigger required for online
+            client to create empy answers for any new study. The offline
+            client does not require this functionality.
+        """
+        familia_data = self.validated_data.pop('familia')
+        respuestas = self.validated_data.pop('respuesta_estudio')
+        instance = save_foreign_relashionship([familia_data], FamiliaSerializer)
+
+        self.validated_data['familia'] = instance[0]
```

Comment:
The relashionship between Familia and Estudio is a One to One Relashionsip. But I will do something to help improve readability.